### PR TITLE
fix: Remove dicer dependency to libs/utils.js

### PR DESCRIPTION
**Pull request by Uzlopak**
based on #102

### CODE REVIEW

File: lib/utils/decode.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
-const { TextDecoder } = require('util')
+let TextDecoder
```
Comment (kibertoad) [inline lib/utils/decode.js:1]: so when do we set this if it's not available?

File: lib/utils/decodeText.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+function decodeText (text, textEncoding, destEncoding) {
+  if (text) {
+    if (textDecoders.has(destEncoding)) {
+      try {
+        return textDecoders.get(destEncoding).decode(Buffer.from(text, textEncoding))
+      } catch (e) { }
+    } else {
+      try {
+        textDecoders.set(destEncoding, new TextDecoder(destEncoding))
```
Comment (Uzlopak) [inline lib/utils/decodeText.js:27]: @kibertoad 
here

File: lib/utils/decode.js (hunk 1 above)
Comment (Uzlopak) [reply lib/utils/decode.js]: see below

File: lib/utils/decodeText.js (hunk 2 above)
Comment (kibertoad) [reply lib/utils/decodeText.js:27]: won't this be undefined in envs without ('util').TextDecoder?

Comment (Uzlopak) [reply lib/utils/decodeText.js:27]: It is in a try catch block and second of all see line 60.

Comment (kibertoad) [reply lib/utils/decodeText.js:27]: ah, right, that makes perfect sense :D

**Review by kibertoad — APPROVED** [review]

**Pull request merged**